src/runtime_src/core/pcie/tools/xbutil/xbutil.cpp

```diff
@@ -908,14 +909,27 @@ int runShellCmd(const std::string& cmd, std::string& output)
 int xcldev::device::runTestCase(const std::string& exe,
     const std::string& xclbin, std::string& output)
 {
-    std::string testCasePath = dsaPath +
+    std::string dirPath;
+    struct stat st;
+
+    output.clear();
+    
+    if (stat(xsaPath.c_str(), &st) != 0) {
+      if (stat(dsaPath.c_str(), &st) != 0) {
```

Comment:
4-space indent please

Reply:
thanks, adjusted it. we still have some cpp code with 2 space indents, I will adjust my .vimrc config.